src/main/java/gameframework/gui/GameMenuBar.java

```diff
@@ -6,14 +6,27 @@
 import java.awt.event.ActionEvent;
 import java.awt.event.ActionListener;
 
+/**
+ * This class represents the menu bar where user can interact with
+ *
+ */
 public class GameMenuBar {
 
+	/**
+	 * Method used to get the component representing a menu bar
```

Comment:
From just the code we can already see that:
- it's a method,
- it gets a component,
- the component is a menu bar